src/extension.ts

```diff
@@ -158,8 +158,8 @@ export function activate(context: vscode.ExtensionContext): void {
         if (await vscode.window.showInformationMessage('Deploy to web app?', yesButton, noButton) === yesButton) {
             this.properties[deployingToWebApp] = 'true';
 
-            const fsPath = await util.showWorkspaceFoldersQuickPick("Select the folder to deploy", this.properties);
-            await createdApp.treeItem.deploy(createdApp, fsPath, outputChannel, ui, reporter, extensionPrefix, false, this.properties);
+            const fsPath = await util.showWorkspaceFoldersQuickPick("Select the folder to deploy", this.properties, configurationSettings.deploySubpath);
+            await createdApp.treeItem.deploy(node, fsPath, outputChannel, ui, reporter, extensionPrefix, false, this.properties);
```

Comment:
I believe this is a bad merge - the two calls in this file should not use 'node' anymore. We just changed this yesterday: https://github.com/Microsoft/vscode-azureappservice/pull/413/files

Reply:
Otherwise this looks good though :)

Reply:
Ok, should be fixed now... sorry for the wait!